convert2rhel/checks.py

```diff
@@ -370,15 +370,32 @@ def ensure_compatibility_of_kmods():
     if not unsupported_kmods:
         logger.debug("All loaded kernel modules are available in RHEL.")
     else:
-        logger.critical(
-            "The following loaded kernel modules are not available in RHEL:\n{0}\n"
-            "First, make sure you have updated the kernel to the latest available version and rebooted the system.\n"
-            "If this message appears again after doing the above, prevent the modules from loading by following {1}"
-            " and run convert2rhel again to continue with the conversion.".format(
-                "\n".join(unsupported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
+
+        not_supported_kmods = "\n".join(
+            map(
+                lambda kmod: "/lib/modules/{kver}/{kmod}".format(kver=system_info.booted_kernel, kmod=kmod),
+                unsupported_kmods,
             )
         )
```

Comment:
Is this really necessary? It's different from upstream, which assumes that anything returned from `get_unsupported_kmods` will be the unsupported kmods, so, this variable seems unnecessary in my point of view. 

Maybe something changed since I last reviewed?

Reply:
I have removed this code and pushed the changes! Thank you for spotting this!